worldedit-core/src/main/java/com/sk89q/worldedit/extent/transform/BlockTransformExtent.java

```diff
@@ -126,7 +126,7 @@ public static <B extends BlockStateHolder<B>> B transform(B block, Transform tra
 
         for (Property<?> property : properties) {
             if (property instanceof DirectionalProperty dirProp) {
-                Direction value = (Direction) result.getState(property);
+                Direction value = (Direction) block.getState(property);
```

Comment:
are you sure these changes are the issue? these were done specifically because it caused only the last property to be modified to actually apply to the resulting block

Reply:
also outside of the case where this is a block that modifies multiple properties with a transform, this code is functionally identical. `B result = block;` is line 124, so for the vast majority of blocks this should be identical

Reply:
The referenced commit is the only one in the 7.3.2 release that seemed suspicious to me. Completely reverting it fixes the issue. This pull request is a smaller revert that still fixes it; not sure why it does, I didn't get that far.

Reply:
`directionalProperties` was getting the wrong value inserted. The regression code was getting the result state rather than the original state.

Reply:
makes sense, thanks